File: PWII/POO/poo.java
public class poo {
    int rodas;
    int portas;
    String cor;
    String marca;
    String modelo;
    String placa;
    String acessorios;
    public poo(int rodas, int portas, String cor, String marca, String modelo, String placa, String acessorios){
        this.rodas = rodas;
        this.portas = portas;
        this.cor = cor;
        this.marca = marca;
        this.modelo = modelo;
        this.placa = placa;
        this.acessorios = acessorios;
    }
    public static void main(String[] args) {
        poo carro = new poo(4, 4, "Azul", "Volkswagen", "SpaceFox", "BRA2E19", "Ar Condicionado, Trava Elétrica");
        System.out.println(carro);
    }
}

File: PWIII/POO/poo.js
class Carro{
    constructor(rodas, porta, cor, marca, modelo, placa, acessorios){
        this.rodas = rodas;
        this.portas = porta;
        this.cores = cor;
        this.marcas = marca;
        this.modelos = modelo;
        this.placas = placa;
        this.acessorio = acessorios;
    }
}
const myCar = new Carro(4, 4, "Dark-Blue", "Honda", "Civic 2000", "BRA2E19", "Ar Condicionada, Caixa de som");
console.log(myCar);
//
console.log(`O MODELO DO CARRO É ${myCar.modelos}`);
console.log(`A marca do carro é ${myCar.marcas}`);
console.log(`A cor do seu ${myCar.modelos} é ${myCar.cores}`);
let pergunta = prompt("DIGITE: ");